operator/bundle_helpers/requirements-upstream.txt

```diff
@@ -0,0 +1,2 @@
+PyYAML==6.0
```

Comment:
I suggest renaming this file to `requirements-gha.txt`. The idea is to move away from "upstream"/"downstream" terminology and division as "downstream" is hopefully going away.

I also suggest adding a `TODO(ROX-26860)` comment meaning we want to delete this file and switch to just `requirements.txt`.